client/keys.go

```diff
@@ -240,6 +282,12 @@ func (k *Key) PublicArea() tpm2.Public {
 	return k.pubArea
 }
 
+// PublicAreaDirect exposes the key's entire direct public area. This is useful for
+// determining additional properties of the underlying TPM key.
+func (k *Key) PublicAreaDirect() tpmt.Public {
```

Comment:
let's unexport this. Otherwise, when we finish the migration, we would have to break uses of `Key.PublicArea` (signature change) as well as `Key.PublicAreaDirect` (removing altogether). Although the first is a required breaking change, we should generally prefer fewer breaking changes.